mmtf-api/src/main/java/org/rcsb/mmtf/api/DataApiInterface.java

```diff
@@ -144,7 +144,7 @@
 
 
 	/**
-	 * Returns the a list length N models, indicating the number of (internal) chains in each model.
+	 * Returns an array of length N models, indicating the number of (internal) chains in each model.
```

Comment:
We need to explain well somewhere (perhaps on the class docs on top of this file) what we mean by internal chains and be consistent with the nomenclature throughout